features/OCM_rule_feedback.feature

```diff
@@ -0,0 +1,240 @@
+     Then the new page should be loaded into web browser
+      And the page title should be the same as rule name
+      And the following tab headers should be displayed
+          | Tab header       | Optional |
+          | How to remediate | no       |
+          | Reason           | no       |
+          | Additional info  | yes      |
+      And "How to remediate" tab should be selected
+      And that page should contain vote "Is this helpful?" with actions
+          | Action     | State      | Style      |
+          | Thumb up   | unselected | BW outline |
+          | Thumb down | unselected | BW outline |
+     When user click on "Thumb up" button
+     Then the message "Thank you for your feedback!" should be displayed
+      And that page should contain vote "Is this helpful?" with actions
+          | Action     | State      | Style         |
+          | Thumb up   | selected   | filled, green |
+          | Thumb down | unselected | BW outline    |
+     When user click on "Thumb up" button
+     Then the message "Thank you for your feedback!" should be displayed
+      And that page should contain vote "Is this helpful?" with actions
+          | Action     | State      | Style      |
+          | Thumb up   | unselected | BW outline |
+          | Thumb down | unselected | BW outline |
+     When user U1 selects the "Log out" item from the user menu
+     Then the page with title "Log in to your Red Hat account" is displayed
+
+
+  Scenario: Downvote for rule on OCM UI
+    Given console.redhat.com is accessible
+      And user U1 is already logged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) with at least one cluster C1
+      And the cluster C1 sent data to the external data pipeline
+      And 1 potential security issue has been identified (ie. one security recommendation has been generated by CCX pipeline)
+      And no voting has been made across the account
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+      And the page should contain at least one cluster
+      And button "Create cluster" should be made visible
+      And button "Register cluster" should be made visible
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "1 potential issue identified"
+      And the page should contains several sections
+      And "Recommendations by category section" should display the following data
```

Comment:
`"Recommendations by category section"` should be `"Recommendations by category" section`.